app/api/helpers/tasks.py

```diff
@@ -7,6 +7,7 @@
 from flask import current_app, render_template
 from flask_celeryext import RequestContextTask
 from marrow.mailer import Mailer, Message
+from marrow.mailer.exc import MessageFailedException, DeliveryFailedException
```

Comment:
You have unused imports.

Reply:
Yeah, I'm focusing on fixing travis first. Will do it.

Reply:
Focus now